.gitignore

```diff
@@ -0,0 +1,278 @@
+
+# Installer logs
+
+# Unit test / coverage reports
+
+# Translations
+
+# Scrapy stuff:
+
+# Sphinx documentation
+
+# PyBuilder
+
+# pyenv
+
+# pipenv
+#   According to pypa/pipenv#598, it is recommended to include Pipfile.lock in version control.
+#   However, in case of collaboration, if having platform-specific dependencies or dependencies
+#   having no cross-platform support, pipenv may install dependencies that don't work, or not
+#   install all needed dependencies.
+
+# celery beat schedule file
+
+# SageMath parsed files
+
+# Spyder project settings
+
+# Rope project settings
+
+# Mr Developer
+
+# mkdocs documentation
+
+# mypy
+
+# Pyre type checker
+
+### venv ###
+# Virtualenv
+# http://iamzed.com/2009/05/07/a-primer-on-virtualenv/
+backend/*/pyvenv.cfg
+backend/*/.env
+backend/*/.venv
+backend/*/env/
+backend/*/venv/
+backend/*/ENV/
+backend/*/env.bak/
+backend/*/venv.bak/
+backend/*/pip-selfcheck.json
+
+### VisualStudioCode ###
+backend/*/.vscode/*
+#backend/*/!.vscode/settings.json
+#backend/*/!.vscode/tasks.json
+#backend/*/!.vscode/launch.json
+#backend/*/!.vscode/extensions.json
+
+### VisualStudioCode Patch ###
+# Ignore all local history of files
+backend/*/.history
+
+# End of https://www.gitignore.io/api/venv,flask,python,visualstudiocode
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# # # # # # # # # # #     End of Backend Python Services    # # # # # # # # # # # #
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+
+
+
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# # # # # # # # # # # # # #    Frontend Angular Apps    # # # # # # # # # # # # # #
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+# Angular generated .gitignore
+
+# Modified with 
+#     `^([^#\n\r/].*)$` => `/$1`
+#     then
+#     `^([^#\n\r].*)$` => `frontend/*$1`
+# at https://regexr.com/
+
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+frontend/*/dist
+frontend/*/tmp
+frontend/*/out-tsc
+# Only exists if Bazel was run
+frontend/*/bazel-out
+
+# dependencies
+frontend/*/node_modules
+
+# profiling files
+frontend/*/chrome-profiler-events*.json
+frontend/*/speed-measure-plugin*.json
+
+# IDEs and editors
+frontend/*/.idea
+frontend/*/.project
+frontend/*/.classpath
+frontend/*/.c9/
+frontend/*/*.launch
+frontend/*/.settings/
+frontend/*/*.sublime-workspace
+
+# IDE - VSCode
+frontend/*/.vscode/*
+#!.vscode/settings.json
+#!.vscode/tasks.json
+#!.vscode/launch.json
+#!.vscode/extensions.json
+frontend/*/.history/*
+
+# misc
+frontend/*/.sass-cache
+frontend/*/connect.lock
+frontend/*/coverage
+frontend/*/libpeerconnection.log
+frontend/*/npm-debug.log
+frontend/*/yarn-error.log
+frontend/*/testem.log
+frontend/*/typings
+
+# System Files
+frontend/*/.DS_Store
+frontend/*/Thumbs.db
```

Comment:
These two should be top level, not just for frontend

Reply:
👍